build-go.sh

```diff
@@ -240,13 +255,19 @@ function buildWithMatrix() {
 	local distname
 	distname=$(basename "$(pwd)")
 
+	local plugin="false"
+	if [ -e plugin ]; then
+	    plugin="true"
+	fi
```

Comment:
Are you manually adding a `plugin` file to indicate something is a plugin? If so could you document it, and if not too painful add it to `dist.sh new-go-dist`? If it's a pain then just documenting should be fine.